column/api/manager/credential_manager.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+from column import utils
+
+
+LOG = logging.getLogger(__name__)
+
+
+class CredentialManager(object):
+    """Column Credential Manager class
+
+    The Credential Manager layer is to support additional logic which is needed
```

Comment:
This sentence doesn't say anything more than noting the names of obvious methods.